openslides_backend/action/actions/meeting/archive.py

```diff
@@ -23,7 +23,18 @@ class MeetingArchive(UpdateAction, GetMeetingIdFromIdMixin):
     schema = DefaultSchema(Meeting()).get_update_schema()
 
     def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        meeting = self.datastore.get(
+            FullQualifiedId(self.model.collection, instance["id"]),
+            ["committee_id"],
+        )
+        committee = self.datastore.get(
+            FullQualifiedId(Collection("committee"), meeting.get("committee_id", 0)),
+            ["organization_id"],
+        )
+        organization_id = committee.get("organization_id")
```

Comment:
You can remove this and just use a constant `1` for the organization, since it is not allowed to have multiple orgas. If we would ever want to implement more organization, we wuld probably run into a lot more problems, so we do not need to consider this case as of now.

Reply:
Okay, done.